tests/integration/ambient/main_test.go

```diff
@@ -95,7 +95,7 @@ values:
   ztunnel:
     terminationGracePeriodSeconds: 5
     env:
-      SECRET_TTL: 5m
+      SECRET_TTL: 1m
```

Comment:
Could this create unexpected behavior in the test assertions due to the recent [cert refresh_at](https://github.com/istio/ztunnel/blob/master/src/identity/manager.rs#L36) update?